compiler/rustc_lint/src/dangling.rs

```diff
@@ -151,6 +153,40 @@ fn lint_expr(cx: &LateContext<'_>, expr: &Expr<'_>) {
     }
 }
 
+/// Determine if a `#[rustc_as_ptr]` receiver type (with adjustments) is only applied
+/// to owned allocations.
+fn is_inner_allocation_owned(input_ty: Ty<'_>, adjs: &[Adjustment<'_>]) -> bool {
+    // 1. Look at all the deref-ed types and exclude any pointer types
+    //
+    //  - `Box<u8>` -> `u8`: OK
+    //  - `Box<&u8>` -> `&u8`: KO
```

Comment:
```suggestion
    //  - `Box<&u8>` -> `&u8`: OK
```

Reply:
`KO` is the correct word, it's the inverse of `OK`; in this context it means that this code should reject it

Reply:
I think this is pretty confusing even if its intention, can we write something else like WARN

Reply:
I have changed them to "WARN" and "NOT WARN".